moby/moby
# daemon: remove support for deprecated `io.containerd.runtime.v1.linux`

**- What I did**

Removed support for `io.containerd.runtime.v1.linux`.

The runtime v1 was deprecated in Docker 20.10.0: f63f73a4a8f531813d6b46a2347cab4bfd210df7 (https://github.com/moby/moby/pull/41182)

**- How I did it**
Removed the corresponding codes and the hack scripts

**- How to verify it**
```console
$ docker run --rm --runtime=io.containerd.runtime.v1.linux hello-world
docker: Error response from daemon: Unknown runtime specified io.containerd.runtime.v1.linux.
See 'docker run --help'.
```

**- Description for the changelog**

daemon: remove support for deprecated `io.containerd.runtime.v1.linux`

**- A picture of a cute animal (not mandatory but encouraged)**
🐧
